dotnet/src/SemanticKernel/Text/TextChunker.cs

```diff
@@ -59,13 +70,16 @@ public static List<string> SplitPlainTextParagraphs(List<string> lines, int maxT
     /// <param name="lines">Lines of text.</param>
     /// <param name="maxTokensPerParagraph">Maximum number of tokens per paragraph.</param>
     /// <param name="overlapTokens">Number of tokens to overlap between paragraphs.</param>
+    /// <param name="tokenCounter">Function to count tokens in a string. If not supplied, the default counter (input length divided by 4) will be used.</param>
```

Comment:
nit: I don't love leaving the implmentation details in the comments as it could change and become out of sync.

Reply:
Good point, removed.